### PR TITLE
[ALLUXIO-2349]Improve error handling in GCSUnderFileSystemFactory#create "@nicefish666"

### PR DESCRIPTION
**Pull request by nicefish666**
https://alluxio.atlassian.net/browse/ALLUXIO-2349
Right now we handle an exception by both logging and rethrowing the exception. This isn't great because the code which catches the thrown exception will also log it, so we will log the same exception multiple times.
We should remove the log statement.

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, add to whitelist

**Comment by aaudiber** [comment]
Jenkins, test this please

File: underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystemFactory.java [hunk 1]
```diff
@@ -55,7 +55,7 @@ public UnderFileSystem create(String path, Object unusedConf) {
     }
 
     String err = "Google Credentials not available, cannot create GCS Under File System.";
-    LOG.error(err);
+
```
Comment (aaudiber) [inline underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystemFactory.java]: could we delete this line entirely?

Comment (nicefish666) [inline underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystemFactory.java]: ok

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
@nicefish666 Thanks for the PR! Could you also sign the CLA agreement?

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by nicefish666** [comment]
I have made a new pr,please closed this pr

**Pull request closed without merging**